pragma-entities/src/models/entries/entry.rs

```diff
@@ -105,9 +105,17 @@ impl Entry {
     pub fn get_last_updated_timestamp(
         conn: &mut PgConnection,
         pair: String,
+        max_timestamp: i64,
     ) -> DieselResult<Option<chrono::NaiveDateTime>> {
+        let Some(max_timestamp) = chrono::DateTime::from_timestamp(max_timestamp, 0) else {
```

Comment:
you can use the new macro `convert_timestamp_to_datetime` here so also support timestamps in ms actually

Reply:
done